locales/en-US/app.ftl

```diff
@@ -102,6 +103,127 @@ ListOfPublishedProfiles--uploaded-profile-information-list =
        *[other] Manage these recordings
     }
 
+## MenuButtons Folder
+
+MenuButtons--index--profile-info-uploaded-label = Uploaded:
+MenuButtons--index--profile-info-uploaded-actions = Delete
+MenuButtons--index--meta-info-subtitle = Profile Information
+MenuButtons--index--meta-info-button-uploaded-profile =
+    .label = Uploaded Profile
+
+MenuButtons--index--meta-info-button-local-profile =
+    .label = Local Profile
+
+MenuButtons--index--full-view = Full View
+MenuButtons--index--cancel-upload = Cancel Upload
+MenuButtons--index--share-upload =
+    .label = Upload
+
+MenuButtons--index--share-re-upload =
+    .label = Re-upload
+
+MenuButtons--index--share-error-uploading =
+    .label = Error uploading
+
+MenuButtons--index--revert = Revert to Original Profile
+MenuButtons--index--docs = Docs
+
+MenuButtons--metaInfo--symbols = Symbols:
+MenuButtons--metaInfo--profile-symbolicated = Profile is symbolicated
+MenuButtons--metaInfo--profile-not-symbolicated = Profile is not symbolicated
+MenuButtons--metaInfo--resymbolicate-profile = Re-symbolicate profile
+MenuButtons--metaInfo--symbolicate-profile = Symbolicate profile
+MenuButtons--metaInfo--attempting-resymbolicate = Attempting to re-symbolicate profile
+MenuButtons--metaInfo--currently-symbolicating = Currently symbolicating profile
+MenuButtons--metaInfo--cpu = CPU:
+
+# Buffer Duration in Seconds in Meta Info Panel
+# Variable:
+#   $physicalCPUs (Number) - Buffer Duration in Seconds
+MenuButtons--metaInfo--physical-cpu =
+    { $physicalCPUs ->
+        [one] { $physicalCPUs } physical core
+       *[other] { $physicalCPUs } physical cores
+    }
+
+# Buffer Duration in Seconds in Meta Info Panel
+# Variable:
+#   $physicalCPUs (Number) - Buffer Duration in Seconds
```

Comment:
`$physicalCPUs` => `$logicalCPUs`

When you'll add the new string as suggested below, don't forget to add a comment too :-)